packages/input-stepper/src/LionInputStepper.js

```diff
@@ -195,61 +235,67 @@ export class LionInputStepper extends LionInput {
   }
 
   /**
-   * Override after template to none
+   * Override before template to none
    * @override
    */
   // eslint-disable-next-line class-methods-use-this
-  _inputGroupAfterTemplate() {
+  _inputGroupBeforeTemplate() {
     return html``;
   }
 
   /**
-   * Override before template to none
-   * @override
+   * Get the decrementor button sign template
+   * @returns {String|import('lit-element').TemplateResult}
    */
   // eslint-disable-next-line class-methods-use-this
-  _inputGroupBeforeTemplate() {
-    return html``;
+  _decrementorSignTemplate() {
+    return '－';
   }
 
   /**
-   * Override prefix template for the increment button
-   * @override
+   * Get the incrementor button sign template
+   * @returns {String|import('lit-element').TemplateResult}
    */
   // eslint-disable-next-line class-methods-use-this
-  _inputGroupPrefixTemplate() {
+  _incrementorSignTemplate() {
+    return '＋';
+  }
+
+  /**
+   * Get the increment button template
+   * @returns {import('lit-element').TemplateResult}
+   */
+  _decrementorTemplate() {
     return html`
       <button
-        ?disabled=${this.disabled || this.readOnly || this.__disableDecrementor}
+        ?disabled=${this.disabled || this.readOnly}
         @click=${this.__decrement}
         tabindex="-1"
         name="decrement"
```

Comment:
What is the need of a `name` attribute on the button?

Reply:
I think it's not needed anymore, I will remove it.